server/routes/IndexRoutes.ts

```diff
@@ -14,15 +14,23 @@ export function setupIndexRoutes(services: NodeServices, router: IRouter) {
   router.get(
     {
       path: API.INDICES_BASE,
-      validate: {},
+      validate: {
+        query: schema.object({
```

Comment:
we can refactor this into a function instead

Reply:
You mean add a function that enhances the validation schema object and use that function everywhere instead of adding the dataSource in all routes?

Reply:
yep!

Reply:
Done